webapp/src/components/table/table.tsx

```diff
@@ -215,7 +215,7 @@ const Table = (props: Props) => {
     const titleSortOption = activeView.sortOptions.find((o) => o.propertyId === Constants.titleColumnId)
     let titleSorted: 'up' | 'down' | 'none' = 'none'
     if (titleSortOption) {
-        titleSorted = titleSortOption.reversed ? 'up' : 'down'
+        titleSorted = titleSortOption.reversed ? 'down' : 'up'
```

Comment:
`reversed: false` indicates descending and `reversed: true` indicated ascending across the sorting logic. So I haven't changed as there's no right or wrong answer as to whether reversed means ascending or descending.

Reply:
Yes, this is driving me nuts. We have 'reversed', 'up/down', 'asc/dsnd'.

But I agree, I think of ascending as "normal" or "up".